### DISCUSSION TITLE
Last.Fm plugin/integration

### DISCUSSION THREAD

**Discussion by viniybob** [pear-devs/pear-desktop#153] [Ideas]
Can devs put last.fm integration to do scrobbles, please?

**Comment by semvis123** [comment]
@viniybob 
#196 this adds the last.fm integration, would love to hear your feedback on it

**Reply by viniybob** [reply]
Hey, @semvis123 how are you?
How can I get the plugin in my app, please? Do you have a download link with the integrated plugin, please?
Thank you so much!

**Reply by semvis123** [reply]
Hi, I'm great, thanks for asking :)

The last.fm plugin is available in the latest (pre) release, https://github.com/th-ch/youtube-music/releases/tag/v1.12.0 

You should be able to enable the plugin like any other plugin (in the menu bar under plugins, and restart the application to apply), when you open the application for the first time with the plugin enabled, you should be greeted with webpage opened in your browser asking to link your account to YouTube music. After that it should just work. 
Let me know if you get it working :)

**Reply by VanGeko** [reply]
any chance to see the lastfm plugin, in the ytm App?
there is a lot of wrong titles, i need to rename.
